src/site/content/en/blog/use-thumbor/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Optimize images with Thumbor
+subhead: Thumbor can be used for free to resize, compress, and transform images on-demand.
+authors:
+  - katiehempenius
+date: 2019-09-20
+hero: hero.jpg
```

Comment:
I think this is the same hero image that was used in the other thumbor post. What do you think about swapping it?

Reply:
Updated ✔️